src/containers/App/App.js

```diff
@@ -54,3 +57,4 @@ class App extends Component {
 }
 
 export default withRouter(App);
+export {App};
```

Comment:
Not knowing js very well, my reading of how this would work is that

```
import App as 1 from './App';
import { App as 2 } from './App';
```

is that 1 is the default export of the higher order component and 2 is just the app component. And I guess you're doing that so that it's easier to test the App component directly. Nifty.

Reply:
Yeah exactly; the `default` export is what shows up if you do `import Blah from source`, while the other export needs to be explicitly destructured. 

This stuff is just to get that little smoke test green